### DISCUSSION TITLE
GitHub Copilot charges have been made during the trial period.

### DISCUSSION THREAD

**Discussion by YohannHe** [community/community#62027] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I applied for the GitHub Copilot 60-day trial service, but I've been charged for the subscription every month. I would like to request a refund.

**Comment by samus-aran** [comment, marked as answer]
Hi @YohannHe 👋 

We now have the option to self-serve refunds for Copilot. Please head over to the [Help with your GitHub account](https://support.github.com/contact/account?flow=refund_copilot_individual) section and save yourself some time.

Note that, in most cases, the Virtual Agent may be able to offer an immediate refund. But, if your concern is not resolved, you can always use the contact form to get in touch with our billing team.
